spec/requests/vms_spec.rb

```diff
@@ -1117,6 +1117,14 @@ def update_raw_power_state(state, *vms)
           expect(response.parsed_body).to include(expected)
         end
 
+        it "in the future" do
```

Comment:
Can you add a test for an invalid date and tests for warn as well?

Reply:
and test what? warn gets sets in the mixin and is tested in core. as with the invalid date. these don't get set until something gets queued.

Reply:
ok